qualtran/bloqs/data_loading/select_swap_qrom_test.py

```diff
@@ -158,3 +164,17 @@ def test_qroam_many_registers():
         log_block_sizes=(log_block_size,),
     )
     qrom.call_graph()
+
+
+@pytest.mark.parametrize('bloq_example', [_qroam_multi_data, _qroam_multi_dim])
```

Comment:
Why isn't this just a regular old bloq_example auto tester? Won't it run more tests?

Reply:
Changed to `auto tester`. Found and fixed another bug in `nth_operation_callgraph` of `QROM`; which was not updated to support the recently added `target_shapes` feature required for `SelectSwapQROM`.